src/app/content/utils/contentManipulation.ts

```diff
@@ -15,12 +15,18 @@ export const isAbsoluteUrl = (url: string) => {
   return aux;
 };
 
-export function fixRelativeURLs(htmlString: string, sourceUrl?: string): string {
+export function fixRelativeURLs(htmlString: string, selectors: string[], sourceUrl?: string): string {
   const domNode = domParser.parseFromString(htmlString, 'text/html');
-  domNode.querySelectorAll('a').forEach((a: HTMLAnchorElement) => {
-    const hrefValue = a.getAttribute('href');
-    if (hrefValue && !isAbsoluteUrl(hrefValue) && sourceUrl) {
-      a.setAttribute('href', fromRelativeUrl(sourceUrl, hrefValue));
+  domNode.querySelectorAll(selectors).forEach((element: HTMLAnchorElement) => {
```

Comment:
you're passing in a selector but each type of element has a different attribute that you're fixing, this seems kinda messy i think we can do better

is there anything wrong with having it always do both images and anchors? you could do them in different loops.

or you could check the element type inside this loop and see if its an anchor change the href and if its an image change the src.

i think i'd rather do the first thing unless there is some reason that causes issues